### ISSUE TITLE
FlaskAdmin not compatible with SQLAlchemy 2.0

### ISSUE THREAD

**Issue by chstolle** [pallets-eco/flask-admin#2348]
FlaskAdmin does not seem compatible with SQLAlchemy 2.0 yet. I'm getting this import error:

```
    from flask_admin.contrib.sqla import ModelView
../../../../../lib/python3.10/site-packages/flask_admin/contrib/sqla/__init__.py:2: in <module>
    from .view import ModelView
../../../../../lib/python3.10/site-packages/flask_admin/contrib/sqla/view.py:18: in <module>
    from flask_admin.contrib.sqla.tools import is_relationship
../../../../../lib/python3.10/site-packages/flask_admin/contrib/sqla/tools.py:11: in <module>
    from sqlalchemy.ext.associationproxy import ASSOCIATION_PROXY
E   ImportError: cannot import name 'ASSOCIATION_PROXY' from 'sqlalchemy.ext.associationproxy' (lib/python3.10/site-packages/sqlalchemy/ext/associationproxy.py)

```

**Comment by POD666** [comment]
Tmp fix in root `__init__.py` file
```
from sqlalchemy.ext import associationproxy

# fix for flask-admin, compatibility with sqla>2
associationproxy.ASSOCIATION_PROXY = (
    associationproxy.AssociationProxyExtensionType.ASSOCIATION_PROXY
)
```

**Comment by POD666** [comment]
Actually seems to be fixed in [here](https://github.com/flask-admin/flask-admin/commit/726e69fe8a6ae7ac29bba0a75da8d011dad7208c). Available in 1.6.1.
Not sure how I missed it 🤔 Maybe because this version is not highlighted in github releases but available in pypi.

**Comment by POD666** [comment]
This issue could be closed
